tf-test-app/Makefile

```diff
@@ -19,8 +19,19 @@
 BUILD_TF := build_tensorflow
 TF := tensorflow-2.7.0-cp39-cp39-linux_x86_64.whl
 
+kontainer-snap: container
+	docker run --runtime=krun -v /opt/kontain/bin/km_cli:/opt/kontain/bin/km_cli -v $(pwd)/tmp:/mnt:rw \
+		--name test-app-k --rm -d -p 5000:5000 --env=KM_MGTPIPE=/tmp/km.sock test-app-k
+	./test.sh
+	-docker exec test-app-k /opt/kontain/bin/km_cli -s /tmp/km.sock -t
+	while [ -n "$(docker ps -f name=test-app-k -q)" ]; do sleep 1; done
+	chmod +rx tmp/kmsnap
+	docker build -t test-app-snap-k -f Dockerfile_snap.kontain .
+	podman pull docker-daemon:test-app-snap-k:latest
```

Comment:
This statement ensures that podman's cached copy of the container is up to date?

Reply:
yes. The point being that after that we can measure the response time using docker or podman